pyHS100/antitheft.py

```diff
@@ -0,0 +1,199 @@
+"""
+pyHS100
+Python library supporting TP-Link Smart Plugs/Switches (HS100/HS110/Hs200).
+
+The communication protocol was reverse engineered by Lubomir Stroetmann and
+Tobias Esser in 'Reverse Engineering the TP-Link HS110':
+https://www.softscheck.com/en/reverse-engineering-tp-link-hs110/
+
+This library reuses codes and concepts of the TP-Link WiFi SmartPlug Client
+at https://github.com/softScheck/tplink-smartplug, developed by Lubomir
+Stroetmann which is licensed under the Apache License, Version 2.0.
+
+You may obtain a copy of the license at
+http://www.apache.org/licenses/LICENSE-2.0
+"""
+
+
+class AntiTheft(object):
```

Comment:
Instead of resurrecting a python2 branch, would it make more sense to convert the downstream code to use python3, if it isn't too great effort?

Anyway, having a proper python2 support (contained in compat.py, similar to https://github.com/requests/requests/blob/master/requests/compat.py) would also be fine.